src/Akka.Logger.Serilog.Tests/ForContextSpecs.cs

```diff
@@ -35,6 +40,63 @@ public ForContextSpecs(ITestOutputHelper helper) : base(Config, output: helper)
             var logClass = typeof(ActorSystem);
 
             _loggingAdapter = Sys.GetLogger<SerilogLoggingAdapter>(logSource, logClass);
+            _defaultLoggingAdapter = Sys.Log;
+        }
+        
+        /// <summary>
+        /// Used to test that https://github.com/akkadotnet/Akka.Logger.Serilog/issues/284 is fixed
+        /// </summary>
+        [Fact]
+        public void ShouldLogMessageWithContextPropertyDefaultLogger()
```

Comment:
Added tests that start with a default `BugLogger` and ensure that the output matches what a user would see with the `SerilogLoggingAdapter` under the same conditions.